js/src/views/Signer/store.js

```diff
@@ -95,7 +95,11 @@ export default class SignerStore {
     this._api.parity
       .localTransactions()
       .then((localTransactions) => {
-        this.setLocalHashes(Object.keys(localTransactions));
+        const keys = Object
+          .keys(localTransactions)
+          .filter((key) => localTransactions[key].status !== 'invalid');
```

Comment:
Why do we skip `invalid` transactions here? I think it would be quite usefull to re-submit transaction cancelled by mistake. Maybe just change the way those are displayed?